scripts/utils/envutils.py

```diff
@@ -0,0 +1,138 @@
+import os
+from pathlib import Path
+
+LOGLEVEL_MAP = {
+    'none': (0, 'LOG_NONE'),
+    'log_none': (0, 'LOG_NONE'),
+    'error': (1, 'LOG_ERROR'),
+    'log_error': (1, 'LOG_ERROR'),
+    'warn': (2, 'LOG_WARN'),
+    'log_warn': (2, 'LOG_WARN'),
+    'warning': (2, 'LOG_WARN'),
+    'log_warning': (2, 'LOG_WARN'),
+    'info': (3, 'LOG_INFO'),
+    'log_info': (3, 'LOG_INFO'),
+    'debug': (4, 'LOG_DEBUG'),
+    'log_debug': (4, 'LOG_DEBUG'),
+    'verbose': (5, 'LOG_VERBOSE'),
+    'log_verbose': (5, 'LOG_VERBOSE'),
+}
```

Comment:
Holy bagels this looks overkill, just use the single words maybe? XD

Reply:
Nah, it works and allows flexibility :3